evm/src/main/java/org/hyperledger/besu/evm/gascalculator/GasCalculator.java

```diff
@@ -581,15 +583,6 @@ default long getMaxRefundQuotient() {
     return 2;
   }
 
-  /**
-   * Maximum Cost of a Transaction of a certain length.
-   *
-   * @param size the length of the transaction, in bytes
-   * @return the maximum gas cost
-   */
-  // what would be the gas for a PMT with hash of all non-zeros
-  long getMaximumTransactionCost(int size);
```

Comment:
This method is unused, the deletion is a simply cleanup and not related to any other refactoring.